modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/v2_0_0/SXPBlueprintUpgradeProcess.java

```diff
@@ -134,8 +135,9 @@ private void _upgradeSearchBarPortlets() throws Exception {
 							StringBundler.concat(
 								"select name, largeValue from ",
 								"PortletPreferenceValue where ",
-								"portletPreferencesId = ",
-								portletPreferencesIdQuoted))) {
+								"portletPreferencesId = ?"))) {
+
+					preparedStatement2.setString(1, portletPreferencesIdQuoted);
```

Comment:
hi @felipelorenz,

I believe it will break the upgrade because it'll quote the quoted variable, so, the query will looks like:
```sql
select name, largeValue 
  from PortletPreferenceValue 
where portletPreferencesId = ''12345''
```
I believe the proper approach is to get portletPreferencesId as a long:

```java
long portletPreferencesId = resultSet.getLong("portletPreferencesId");

try (PreparedStatement preparedStatement2 =
		connection.prepareStatement(
			StringBundler.concat(
				"select name, largeValue from PortletPreferenceValue where portletPreferencesId = ?"))) {

	preparedStatement2.setLong(1, portletPreferencesId);
```

Reply:
Could you please fix this and put the changes in another commit?

feel free to contact me,

Thank you!

Reply:
@felipelorenz take a look at this [WIP](https://github.com/felipelorenz/liferay-portal/pull/17/commits/b765b9feef9b4d0bacf8243e5bc4cf45a97444b5)